docs/advanced/custom-renderer.md

```diff
@@ -193,11 +152,79 @@ export const CustomTemplate: FunctionComponent<
 };
 ```
 
-### Completion
+Now that the component has been created, we can add a story to view it. Next to `src/templates/coc/template.tsx` create a file called `template.stories.mdx` with the following content:
+
+```markdown
+import { Meta, Preview, Props, Description, Story } from "@storybook/addon-docs/blocks";
+import { object } from "@storybook/addon-knobs";
+import { CocTemplate } from "./template";
+import { cocTemplateCertificate } from "../sample";
+
+<Meta title="MDX|CocTemplate" component={CocTemplate} />
+
+# CocTemplate component
+
+<Description of={CocTemplate} />
+
+# Props
+
+<Props of={CocTemplate} />
+
+# Usage
+
+<Preview>
+  <Story name="basic sample">
+    <CocTemplate document={object("document", cocTemplateCertificate)} />
+  </Story>
+</Preview>
+```
+
+We can now [start storybook](#run-development-preview) and make sure our component looks like expected.
 
 ![Completed Story Book View](/docs/advanced/custom-renderer/completed-storybook.png)
 
-Once that template code has been updated, you will see that the storybook component has been updated to reflect the change.
+### Certificate of Completion template configuration
+
+An OA document may have multiple views, each of them rendered in separate tabs. For example, an OA document that is a degree certificate may have the actual certificate as one view, and the transcript as another view in a single template. A demo of the multiple views feature can be found [here](https://opencerts.io/?q={%22type%22:%22DOCUMENT%22,%22payload%22:{%22uri%22:%22https://opencerts.io/static/demo/mainnet.opencerts%22,%22permittedActions%22:[%22STORE%22],%22redirect%22:%22https://opencerts.io%22}}).
```

Comment:
I believe this link was changed from `https://opencerts.io/static/demo/mainnet.opencerts` to `https://opencerts.io/static/demo/homestead.opencert`

Reply:
oh yeah good catch

Reply:
fixed :)